src/test/test-ask-password-api.c

```diff
@@ -10,10 +10,13 @@ static void ask_password(void) {
         _cleanup_strv_free_ char **ret = NULL;
 
         r = ask_password_tty(-1, "hello?", "da key", 0, 0, NULL, &ret);
-        assert(r >= 0);
-        assert(strv_length(ret) == 1);
-
-        log_info("Got %s", *ret);
+        if (r == -ECANCELED)
+                assert_se(ret == NULL);
+        else {
+                assert(r >= 0);
+                assert(strv_length(ret) == 1);
+                log_info("Got \"%s\"", *ret);
```

Comment:
Is it safe to log this information?

Reply:
Yeah, that's just for testing.

Reply:
But there should be assert_se here too. I pushed an updated version.